flask_mongorest/resources.py

```diff
@@ -509,6 +509,28 @@ def value_for_field(self, obj, field):
         """
         raise UnknownFieldError
 
+    def _rename_dict(self, data):
+        self._rename_dict_with_mapping(data, self._rename_fields)
+
+    def _reverse_rename_dict(self, data):
+        self._rename_dict_with_mapping(data, self._reverse_rename_fields)
+
+    @staticmethod
+    def _rename_dict_with_mapping(data, mapping):
+        # Do renaming in two passes to prevent potential multiple renames
+        # depending on dict traversal order.
+        # E.g. if a -> b, b -> c, then a should never be renamed to c.
+        fields_to_delete = []
+        fields_to_update = {}
+        for k, v in mapping.items():
```

Comment:
all the `k`s and `v`s in this method could be replaced with more accurate names describing what these variables actually hold.